server/routes/api/stats.py

```diff
@@ -704,3 +846,24 @@ def get_stats_within_place_csv():
             parent_place, child_type)
     response.status_code = 200
     return response
+
+
+@bp.route('/api/stats/csv/within-place', methods=['POST'])
+def get_stats_within_place_csv():
+    """Gets the statistical variable data as a csv for child places of a
+    certain place type contained in a parent place. If no date range specified,
+    gets data for all dates of a series. If minDate and maxDate are "latest",
+    the latest date data will be returned.
+
+    Request body:
+        parentPlace: the parent place of the places to get data for
+        childType: type of places to get data for
+        statVars: list of statistical variables to get data for
+        minDate (optional): earliest date to get data for
+        maxDate (optional): latest date to get data for
+        facetMap (optional): map of statistical variable dcid to the id of the
+            facet to get data from
+        rowLimit (optional): number of csv rows to return
+    """
+    req_str = json.dumps(request.json, sort_keys=True)
+    return get_stats_within_place_csv_wrapper(req_str)
```

Comment:
Can remove this wrapper now since dc.xxx call is cached. Also no need to cache this route either.

Reply:
get_stats_within_place_csv_wrapper has a decent amount of computation tho .. do we not want to cache that?

Reply:
I am not quite sure on this, for following reasons

1) The request body has a lot of params, it might not be common for a lot of requests to have the same request json (different users download exact the same thing)
2) The computation is mostly data conversion, which is fairly cheap

I feel caching makes sense if it involves API call (network traffic) or the request is very common and could be repeatedly used (browser/place page). Download tool might not be like that...

Reply:
Another reason is we don't want to cache all the stuff, as the memcache is getting full all the time already.. so keys are being evicted. https://pantheon.corp.google.com/memorystore/redis/locations/us-central1/instances/webserver-cache/details?project=datcom-website-prod

Reply:
ah, ok, that makes sense to me, I will remove the caching for this one!

Reply:
done